cypress.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Do you need it?

Reply:
`cypress` adds it on running, I don't want to gitignore it because it is supposed to be used for overrides ...